pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -1476,6 +1476,17 @@
         "4.0.3.0": "4.4.0"
       }
     },
+    "System.Memory": {
+      "StableVersions": [
```

Comment:
This does not sound right. There is no stable 4.0.0 version of this package... .